openai/openai.go

```diff
@@ -175,7 +175,16 @@ func toListCompletion(r api.ListResponse) ListCompletion {
 	}
 }
 
-func fromRequest(r ChatCompletionRequest) api.ChatRequest {
+func toRetrieveCompletion(r api.ShowResponse, model string) Model {
```

Comment:
should we change this to `toModel`? 

```suggestion
func toModel(r api.ShowResponse, model string) Model {
```